AGPs/AGP-FutarchySignaling.md

```diff
@@ -0,0 +1,64 @@
+---
+AGP: N/A
+Title: Futarchy Signaling Markets for AGP Votes
+Author: John Kelleher (@jpkcambridge)
+Status: Stage III
+Track: Finance
+Created: 2019-04-06
+---
+
+## Address of the transfer recipient
+
+TBD
+
+## Amount of the transfer
+
+30,000 DAI
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1
+
+## Purpose of the transfer
+
+"Aragon is a fight for freedom.  Aragon empowers freedom by creating liberating tools that leverage decentralized technologies." - [AGP-0, The Aragon Manifesto](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-0.md)
+
+[Futarchy](http://mason.gmu.edu/~rhanson/futarchy.html), described as "Vote your Values, Bet Your Beliefs," offers a [collusion resistant](https://www.vitalik.ca/general/2019/04/03/collusion.html) way for organizations to make decisions. By using prediction markets, participants are incentivized to evaluate a decision based on a given metric.  For organizations aligned around given values, futarchy offers the promise of enabling well informed decisions without delegating power to the few.  Futarchy offers the potential to be a liberating tool, and advancements in decentralized technology are allowing it to come to fruition.  We seek to demonstrate and improve upon an implementation of futarchy, better understand futarchy in practice, and to educate the Aragon network and the public at large on the promise of Futarchy.
```

Comment:
> to educate the Aragon network and the public at large on the promise of Futarchy

Is "futarchy" supposed to be capitalized here? It seems to be spelled lower-cased elsewhere in the doc.